### PR TITLE
Adding Ubuntu binaries (#3693)

### LINKED ISSUE #3693: Ubuntu 16.04 binaries
Add the following binaries for Ubuntu 16.04:

* 2.0.0-p648
* 2.1.9
* 2.2.5
* 2.3.0

### PR DESCRIPTION
**Pull request by raelgc**
Fixes #3693

### CODE REVIEW

File: config/md5 [hunk 1]
```diff
@@ -1060,6 +1060,11 @@ https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.0.0-p59
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.1.5.tar.bz2=57bd1de4797ef8c4b151622a47a10e4a
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.2.0.tar.bz2=e68c9d66cf6309e05056b8e381a9f05c
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.2.1.tar.bz2=30ddde33731e10c38e4857384edca8f2
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.0.0-p648.tar.bz2=
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.9.tar.bz2=
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.2.5.tar.bz2=
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.3.0.tar.bz2=928f951e46c85c533fd5f1f6e1210a2f
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.3.1.tar.bz2=
```
Comment (mpapis) [inline config/md5]: where are the md5's? a bug in a script?

**Comment by raelgc** [comment]
@mpapis Bug in the operator, fixed now :D

**Comment by raelgc** [comment]
@rys or @mpapis Can you review and merge, please?

**Comment by mpapis** [comment]
Thanks

**Pull request merged**